distribution/lib/Standard/Base/0.0.0-dev/src/Meta.enso

```diff
@@ -14,10 +14,28 @@ import project.Nothing.Nothing
 import project.Polyglot.Java
 
 import project.Error.Error as Base_Error
-import project.Polyglot.Polyglot as Base_Polyglot
+import project.Polyglot as Base_Polyglot
 
 from project.Data.Boolean import Boolean, True, False
 
+type Type
+    ## UNSTABLE
+       ADVANCED
+
+       Type meta-representation.
+
+       Arguments:
+       - value: The value of the type in the meta representation.
+    Value value
+
+    ## UNSTABLE
+       ADVANCED
+
+       Returns a vector of field names defined by a constructor.
+    constructors : Vector
+    constructors  self =
+        Vector.from_polyglot_array (get_type_constructors self.value) . map Meta.meta
```

Comment:
```suggestion
        Vector.from_polyglot_array (get_type_constructors self.value) . map Constructor.Value
```

We know it's an Atom constructor, maybe we can try this?

Not sure if it will prevent the thing from materializing though.

We may need to use the `...` operator carefully around here, but I'm not certain really how - this operator is very tricky for me - when it's needed and when not.

Reply:
Using `Constructor.Value` doesn't work. Yeah `...` is probably needed - but how!?

Reply:
Resolved with [1b89cad](https://github.com/enso-org/enso/pull/5956/commits/1b89cadf069bbd5b7c433fa8ea62e7badb57874e).